generators/app/templates/src/theme/__ionic.theme-variables.scss

```diff
@@ -32,54 +26,110 @@ $app-direction: ltr;
 // so you can add, rename and remove colors as needed.
 // The "primary" color is the only required color in the map.
 
-$colors: (
-  primary:    #488aff,
-  secondary:  #32db64,
-  danger:     #f53d3d,
-  light:      #f4f4f4,
-  dark:       #333
-);
+:root {
```

Comment:
is `:root` really necessary here?

Reply:
tbh I'm not sure, but their theming guides say to use :root, and their starters have it too.